src/applications/disability-benefits/686c-674/config/chapters/household-income/householdIncome.js

```diff
@@ -8,6 +8,7 @@ export const uiSchema = {
   householdIncome: {
     'ui:options': {
       hideIf: () => environment.isProduction(),
+      hideOnReview: () => environment.isProduction(),
```

Comment:
Do you know if the other questions that were hidden in the previous PR also need the `hideOnReview` flag set? Or will they not show up on the review page?

Reply:
@micahchiang I checked and the questions that are embedded in other pages hide fine on the review pages, it was just these ones that needed to be hid on review because they are in their own page